tidy3d/web/core/system_config.py

```diff
@@ -0,0 +1,42 @@
+"""Tidy3d system config."""
+
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic.v1 import Extra, Field
+
+from .http_util import http
+from .types import Tidy3DResource
+
+
+class SystemConfig(Tidy3DResource, extra=Extra.allow):
+    """Tidy3D system config."""
+
+    end_statuses: Optional[tuple] = Field(
+        None,
+        title="endStatuses",
```

Comment:
title="End Statuses"
description="Tuple of status keys that signify that the task has completed."